receiver/otlpreceiver/otlp.go

```diff
@@ -180,15 +205,21 @@ func (r *otlpReceiver) Shutdown(ctx context.Context) error {
 		if r.serverGRPC != nil {
 			r.serverGRPC.GracefulStop()
 		}
+
+		r.shutdownWG.Wait()
 	})
 	return err
 }
 
-func (r *otlpReceiver) registerTraceConsumer(ctx context.Context, tc consumer.TracesConsumer) error {
+func (r *otlpReceiver) registerTraceConsumer(tc consumer.TracesConsumer) error {
```

Comment:
Please split this in a separate PR.

Reply:
With the non-overlapping lifecycle, this is not needed anymore, will remove.